### PR TITLE
small fix

### PR DESCRIPTION
**Pull request by EbrahimKreem**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by EbrahimKreem** [comment]
Screenshot from 2019-09-30 00-05-24

File: projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
 const path = require('path');
 // @TODO update require. when we export this method - we can connect it from index.js
-const {
-  parseDirectoryFiles,
-  getHeaders
-} = require('../../../../dist/index.cjs');
+const { parseDirectoryFiles, parseCsv } = require('../../../../dist/index.cjs');
```
Comment (atherdon) [inline projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: maybe we can also fix problem with this path?
it drives me crazy

File: projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 
 async function Main() {
-  const headers = await getHeaders('Fish_Reftbl_RefDatasets.csv');
+  const headers = await parseCsv(`${__dirname}/Fish_Reftbl_RefDatasets.csv`);
```
Comment (atherdon) [inline projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: if now we don't use getHeaders in a streight way - maybe it better to remove it's export at index.js?

Comment (atherdon) [reply projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: another request - can you also make same change at other parsers, located at FAO folder?

File: projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js (hunk 1 above)
Comment (EbrahimKreem) [reply projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: we can solve this with `app-root-path` package but the code in parser will be like this

`const appRoot = require('app-root-path');

const { parseDirectoryFiles, parseCsv } = require(appRoot + '/dist/index.cjs');`

Comment (atherdon) [reply projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: hmm. you make a decision

Comment (EbrahimKreem) [reply projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js]: do you think it will confuse other interns or no?

**Comment by EbrahimKreem** [comment]
:confused: when I try to run the parser 

Screenshot from 2019-09-30 15-53-31

**Comment by EbrahimKreem** [comment]
yesterday it was working

**Comment by atherdon** [comment]
ok, i think it something similar to prev bug that we encountered.

can you still submit commits, related to changes at other parses and about my other comments as well that is pending? I'll take a look utils issue

**Comment by atherdon** [comment]
you know what you can do as quick check?
try to rollback `generator` module to version 1.0.0 and re-install it.
curious to see if that error will still appear

**Comment by EbrahimKreem** [comment]
> try to rollback generator module to version 1.0.0 and re-install it.

yeah I think it's generator module bug when I installed version 1.0.0 everything works as expected

**Comment by atherdon** [comment]
cool, then I should look at utils in generator, not in our csv parser.

please add it to this PR, so we can move forward for this moment

>

**Comment by atherdon** [comment]
@zhangted check this comment = it might help https://github.com/GroceriStar/food-datasets-csv-parser/pull/175#issuecomment-536603369

**Comment by atherdon** [comment]
can you sync your fork and delete yarn.lock then - because it can 

Ebrahim - should I merge this PR or wait when you'll finish working on pending tasks?
i can elaborate what i mean by pending tasks

**Comment by atherdon** [comment]
merging your changes, next thing can be done in next PR

**Pull request merged**